pilot/pkg/features/pilot.go

```diff
@@ -178,6 +178,12 @@ var (
 		"If enabled, metadata exchange will be enabled for TCP using ALPN and Network Metadata Exchange filters in Envoy",
 	).Get()
 
+	EnableWasm = env.RegisterBoolVar(
+		"PILOT_ENABLE_WASM",
+		false,
+		"If enabled, Wasm-based telemetry will be enabled.",
```

Comment:
nit: `ENABLE_WASM_TELEMETRY`. This should be specific to our telemetry, not Wasm in general.

Reply:
sure!